console/src/app/pages/app-quick-create/app-quick-create.component.ts

```diff
@@ -0,0 +1,325 @@
+import { Component, OnDestroy, signal } from '@angular/core';
+import { ActivatedRoute, Navigation, Params, Router } from '@angular/router';
+import { BehaviorSubject, Subject, takeUntil } from 'rxjs';
+import { InfoSectionType } from 'src/app/modules/info-section/info-section.component';
+import { ProjectType } from 'src/app/modules/project-members/project-members-datasource';
+import { ProjectAutocompleteType } from 'src/app/modules/search-project-autocomplete/search-project-autocomplete.component';
+import {
+  AddProjectRequest,
+  AddProjectResponse,
+  AddOIDCAppRequest,
+  AddProjectRoleRequest,
+  AddUserGrantRequest,
+} from 'src/app/proto/generated/zitadel/management_pb';
+import { GrantedProject, Project } from 'src/app/proto/generated/zitadel/project_pb';
+import { Breadcrumb, BreadcrumbService, BreadcrumbType } from 'src/app/services/breadcrumb.service';
+import { ManagementService } from 'src/app/services/mgmt.service';
+import { Framework } from 'src/app/components/quickstart/quickstart.component';
+import frameworkDefinition from '../../../../../docs/frameworks.json';
+import { NavigationService } from 'src/app/services/navigation.service';
+import { Location } from '@angular/common';
+import { ToastService } from 'src/app/services/toast.service';
+import { GrpcAuthService } from 'src/app/services/grpc-auth.service';
+import { OIDC_CONFIGURATIONS } from 'src/app/utils/framework';
+import { generateRandomProjectName, generateFrameworkAppName } from 'src/app/utils/name-generator';
+import { ThemeService } from 'src/app/services/theme.service';
+import { OIDCAppType, OIDCAuthMethodType } from 'src/app/proto/generated/zitadel/app_pb';
+import { MatDialog } from '@angular/material/dialog';
+import { AppSecretDialogComponent } from 'src/app/pages/projects/apps/app-secret-dialog/app-secret-dialog.component';
+
+@Component({
+  selector: 'cnsl-app-quick-create',
+  templateUrl: './app-quick-create.component.html',
+  styleUrls: ['./app-quick-create.component.scss'],
+  standalone: false,
```

Comment:
We have decided to only do standalone: false, also changedection onpush should be used